src/pcgnslib.c

```diff
@@ -421,11 +421,19 @@ int cgp_open(const char *filename, int mode, int *fn)
 
     /* Flag is true if MPI_Init or MPI_Init_thread has been called and false otherwise. */
     pcg_mpi_initialized = 0;
-    /* check if we are actually running a parallel program */
-    MPI_Initialized(&pcg_mpi_initialized);
-
-    /* Flag this as a parallel access */
-    strcpy(hdf5_access,"PARALLEL");	
+    if (mode < 10) {
```

Comment:
Should use the variable instead of a hard-coded number.

Reply:
Good point. Let me fix that